app/views/content/train-to-be-a-teacher/_questions.html.erb

```diff
@@ -24,7 +24,7 @@
         Find out what you'll learn, what classroom experience you'll get and how you'll be assessed.
       </p>
 
-      <a href="/train-to-be-a-teacher/initial-teacher-training" class="button">Find out about initial teacher training</a>
+      <a href="/initial-teacher-training" class="button">Find out about initial teacher training</a>
```

Comment:
@peteryates I think it's the actual page URL that needs amending, not the path on this button - we need to keep with the URLs that we confirmed to the marketing team (see https://trello.com/c/MQgtSwLn)

So this initial teacher training page needs to have a path of /train-to-be-a-teacher/initial-teacher-training ?

Reply:
Ah! Didn't realise, will amend

Reply:
> So this initial teacher training page needs to have a path of /train-to-be-a-teacher/initial-teacher-training

@gemmadallmandfe - this is where I added the ITT card 🙈